en/core-libraries/components/pagination.rst

```diff
@@ -90,6 +90,11 @@ array after the model you wish to configure::
 The values of the ``Post`` and ``Author`` keys could contain all the properties
 that a model/key less ``$paginate`` array could.
 
+.. note::
+    The ``$paginate`` variable is not automatically passed to the :php:class:`PaginatorComponent`'s settings. You must explicitly set the :php:class:`PaginatorComponent`'s settings using the following line::
+
+        $this->Paginator->settings = $this->paginate;
```

Comment:
ps: are you sure that $this->Paginator->settings actually exists? In 3.x usually the config() trait method uses the protected _config attribute.